src/main/java/com/ivanmostovyi/demo/controller/MessagesController.java

```diff
@@ -0,0 +1,47 @@
+package com.ivanmostovyi.demo.controller;
+
+import com.ivanmostovyi.demo.domain.Messages;
+import com.ivanmostovyi.demo.domain.User;
+import com.ivanmostovyi.demo.dto.MessagesFormDto;
+import com.ivanmostovyi.demo.service.MessageService;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
```

Comment:
avoid `*` imports.
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports